atk.yml

```diff
@@ -73,7 +73,7 @@ services:
         IMAGE_BASE: "dustynv/ros"
         IMAGE_TAG: "humble-pytorch-l4t-r35.3.1"
         ROS_INSTALL_PREFIX: "/opt/ros/${ROS_DISTRO}/install"
-        APT_DEPENDENCIES: "${DEFAULT_APT_DEPENDENCIES}"
+        APT_DEPENDENCIES: "${DEFAULT_APT_DEPENDENCIES} ros-${ROS_DISTRO}-robot-localization"
```

Comment:
sorryyyy i miss spoke earlier. What you should do is put dependencies in the package that requires it. So in `art_localization_launch` or whatever, add an exec_depend for `robot_localization` like [here](https://github.com/uwsbel/autonomy-research-testbed/blob/master/workspace/src/perception/cone_detector/package.xml). Does that make sense? Looks good to me when that's done.

Reply:
make sense, but how do i install it within the container? shouldn't it be a part of the container building process?

Reply:
this is the recommended way to add ROS dependencies for ros packages. And so you can install it directly in the container using `rosdep` yourself, or if you're using the atk/docker build system, when building the image, `rosdep` is called automatically. So if you're using atk, you can just rebuild the image and it will be included. make sense?